ts/screens/authentication/UnrecognizedCieScreen.tsx

```diff
@@ -0,0 +1,68 @@
+import { Container, H1, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+type Props = OwnProps;
```

Comment:
```suggestion
type Props = OwnProps &
  Readonly<{
    onRetry: () => void;
  }>;
```
we should provide a prop to provide the onRetry press button action